tests/outputs/TestDiagram.cpp

```diff
@@ -159,8 +155,8 @@ TEST(Diagram, writeEmpty) {
   std::string directoryPath = outputDirectory.generic_string();
   try {
     std::ifstream stream(directoryPath);
-    ASSERT_TRUE(stream.fail()) << "The Diagram directory " << directoryPath << " has been created to even though there are no generators which model requires it."
-                               << std::endl;
+    ASSERT_TRUE(stream.fail()) << "The Diagram directory " << directoryPath
+                               << " has been created to even though there are no generators which model requires it." << std::endl;
```

Comment:
Let's correct this as we modify this part of the code: even though it is not used by any generator model

Reply:
OK